db/re/status.yml

```diff
@@ -8883,3 +8883,79 @@ Body:
       NoBanishingBuster: true
       NoDispell: true
       NoClearance: true
+  - Status: Talisman_of_Protection
+    Icon: EFST_TALISMAN_OF_PROTECTION
+    DurationLookup: SOA_TALISMAN_OF_PROTECTION
+    Flags:
+      BlEffect: true
+      DisplayPc: true
```

Comment:
The `SC` already has these conditions
```
// If the caster is offline, dead, on another map or
// if the target is not a player or is in another party
// End the status change
```
but maybe we could also add the flag
```
	RemoveOnMapWarp       - Removed when warping to another map.
```
?

I guess the `NoSave` flag should also be defined, but that's just a guess
```
	NoSave                - Won't be saved when player logs out.
```